index.php

```diff
@@ -7,30 +7,45 @@
 <main id="main-content">
 
   <!-- main posts loop -->
-  <section id="posts">
+  <section id="posts" class="container">
+    <div class="row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
-?>
 
-    <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+    $single_row = get_post_meta($post->ID, '_igv_single_row', true);
+    $top_margin = get_post_meta($post->ID, '_igv_top_margin', true);
+    $percent_width = get_post_meta($post->ID, '_igv_percent_width', true);
+    $percent_rotate = get_post_meta($post->ID, '_igv_percent_rotate', true);
+
+    $single_row = empty($single_row) ? '' : $single_row;
+    $top_margin = empty($top_margin) ? '0' : $top_margin;
+    $percent_width = empty($percent_width) ? '100' : $percent_width;
+    $percent_rotate = empty($percent_rotate) ? '0' : $percent_rotate;
```

Comment:
fine to keep as percent (tho maybe degrees are just as good for UX) but needs maths here to convert